src/interfaces/facades/IBlobsFacade.sol

```diff
@@ -29,4 +88,32 @@ interface IBlobsFacade {
     /// @param size Blob size.
     /// @param bytesReleased Network capacity bytes released.
     event BlobDeleted(address indexed subscriber, bytes32 hash, uint256 size, uint256 bytesReleased);
+
+    /// @dev Add a new blob to storage.
+    /// @param params Parameters for adding the blob including sponsor, source, hashes, size and TTL.
+    function addBlob(AddBlobParams memory params) external;
+
+    /// @dev Get information about a specific blob.
+    /// @param blobHash Blob blake3 hash.
+    /// @return blob Information, including its hash, size, metadata hash, subscribers and status.
+    function getBlob(bytes32 blobHash) external view returns (Blob memory blob);
+
+    /// @dev Delete a blob from storage.
+    /// @param subscriber The address of the subscriber.
+    /// @param blobHash Blob blake3 hash to delete.
+    /// @param subscriptionId Identifier used to differentiate blob additions for the same subscriber.
+    function deleteBlob(address subscriber, bytes32 blobHash, string memory subscriptionId) external;
```

Comment:
nit: the events use `hash`, not `blobHash`. i don't have a pref either way really... generally if the context is clear enough, i'd omit `blob` but often more explicit is better. whatever we do, i think they should match